app/Mail/InventoryAlertMail.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace App\Mail;
+
+use AllowDynamicProperties;
+use Illuminate\Bus\Queueable;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Mail\Mailable;
+use Illuminate\Mail\Mailables\Address;
+use Illuminate\Mail\Mailables\Content;
+use Illuminate\Mail\Mailables\Envelope;
+use Illuminate\Queue\SerializesModels;
+
+#[AllowDynamicProperties] class InventoryAlertMail extends Mailable
```

Comment:
Interesting...did PHPStorm prompt you to add `AllowDynamicProperties`?

(If so, can you put it on the line above `class`? That's the normal formatting for attributes.)

Reply:
done 👍